.appveyor.yml

```diff
@@ -1,36 +1,45 @@
-build: off
+build: false
 version: dev-{build}
 shallow_clone: false
 clone_folder: C:\projects\php-cs-fixer
 
 environment:
     matrix:
         - php_ver: 5.6.17
+          SKIP_LINT_TEST_CASES: 1
         - php_ver: 7.0.9
 
 cache:
     - '%APPDATA%\Composer'
+    - '%LOCALAPPDATA%\Composer'
+    - C:\tools\php -> .appveyor.yml
 
-install:
-    - choco install -my --allow-empty-checksums php -version %php_ver%
+init:
     - SET PATH=C:\tools\php;%PATH%
+
+install:
+    - IF NOT EXIST C:\tools\php (choco install --yes --allow-empty-checksums php --version %php_ver%)
     - cd C:\tools\php
     - copy php.ini-production php.ini
     - echo date.timezone="UTC" >> php.ini
     - echo memory_limit=512M >> php.ini
     - echo extension_dir=ext >> php.ini
     - echo extension=php_curl.dll >> php.ini
     - echo extension=php_openssl.dll >> php.ini
-    - cd C:\tools
+    - cd C:\projects\php-cs-fixer
     - appveyor DownloadFile https://getcomposer.org/composer.phar
-    - appveyor DownloadFile https://phar.phpunit.de/phpunit-5.7.phar
     - git config --global github.accesstoken 5e7538aa415005c606ea68de2bbbade0409b4b8c
+    - IF EXIST %APPDATA%\Composer\composer.json (
+          php composer.phar global show -ND | findstr "hirak/prestissimo" || php composer.phar global require hirak/prestissimo
+      ) ELSE (
+          php composer.phar global require hirak/prestissimo
+      )
```

Comment:
all 5 lines could be replaced with:
`php composer.phar global show -ND 2>1 | findstr "hirak/prestissimo" || php composer.phar global require hirak/prestissimo`

(`2>1` is added)